config/prod/kelsey-ami.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v8.j2
```

Comment:
there is a managed-ec2-v9.j2 now please review the example template and use that.  thanks.

Reply:
the [example pull request is still v8](https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/3)